scripts/policy/protocols/dns/detect-external-names.zeek

```diff
@@ -39,3 +39,24 @@ event dns_A_reply(c: connection, msg: dns_msg, ans: dns_answer, a: addr) &priori
 		        $identifier=cat(a,ans$query)]);
 		}
 	}
+
+# exactly the same as above.
```

Comment:
If it's exactly the same, might make sense to promote the original handler to a function that's then called from either.